keel-titus-api/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/TitusClusterSpec.kt

```diff
@@ -54,6 +56,45 @@ data class TitusClusterSpec(
   override val unhappyWaitTime: Duration? = null
 ) : ComputeResourceSpec, Monikered, Locatable<SimpleLocations>, UnhappyControl {
 
+  @Factory
+  constructor(
+    moniker: Moniker,
+    deployWith: ClusterDeployStrategy = RedBlack(),
+    locations: SimpleLocations,
+    container: ContainerProvider,
+    capacity: Capacity?,
+    constraints: TitusServerGroup.Constraints?,
+    env: Map<String, String>?,
+    containerAttributes: Map<String, String>?,
+    resources: ResourcesSpec?,
+    iamProfile: String?,
+    entryPoint: String?,
+    capacityGroup: String?,
+    migrationPolicy: TitusServerGroup.MigrationPolicy?,
+    dependencies: ClusterDependencies?,
+    tags: Map<String, String>?,
+    overrides: Map<String, TitusServerGroupSpec> = emptyMap()
+  ) : this(
+    moniker = moniker,
+    deployWith = deployWith,
+    locations = locations,
+    _defaults = TitusServerGroupSpec(
+      capacity = capacity,
+      capacityGroup = capacityGroup,
+      constraints = constraints,
+      dependencies = dependencies,
+      entryPoint = entryPoint,
+      env = env,
+      containerAttributes = containerAttributes,
+      iamProfile = iamProfile,
+      migrationPolicy = migrationPolicy,
+      resources = resources,
+      tags = tags
+    ),
+    overrides = overrides,
+    container = container
+  )
```

Comment:
`@Factory` constructor for `TitusClusterSpec` instead of requiring a custom deserializer